RouletteApplicationServer/src/communication/Client.java

```diff
@@ -2,25 +2,31 @@
 
 import com.kubasz561.roulette.common.JSONMessage;
 
-import java.util.ArrayList;
-
 /**
  * Created by sackhorn on 20.05.17.
  */
 public class Client {
     public ServerCommunicationThread thisClientComThread;
     public int clientId;
+    public String login;
     public String passHash;
     public double accountBalance;
 
     public Client(JSONMessage logInMessage)
     {
-        //TODO:Parse json login message to fields of this class
+        logInMessage.parseJSONString(logInMessage.getRawJSONString());
```

Comment:
Tutaj będzie jakaś inba z merdżem bo modyfiowałem to ale to raczej lajtowo ja rozwiąże konfilkt na twoim branchu jak będziesz chciał merdżować